Source/ThirdParty/skia/CMakeLists.txt

```diff
@@ -895,6 +901,19 @@ add_library(Skia STATIC
     modules/skcms/src/skcms_TransformSkx.cc
 )
 
+if (is_android)
+    target_sources(Skia PRIVATE src/ports/SkDebug_android.cpp)
+    target_sources(Skia PRIVATE src/ports/SkFontMgr_android.cpp)
+    target_sources(Skia PRIVATE src/ports/SkFontMgr_android_parser.cpp)
+
+    target_link_libraries(Skia PRIVATE
+        expat::expat
+    )
+else ()
+    target_sources(Skia PRIVATE src/ports/SkDebug_stdio.cpp)
+    target_sources(Skia PRIVATE src/ports/SkFontMgr_fontconfig.cpp)
```

Comment:
I think we need to add a `target_link_libraries(Skia PRIVATE Fontconfig::Fontconfig)` here, and remove it from the list of libraries right below after the `endif`... As far as I remember Android does not use Fontconfig, or at least it's not exposed to the NDK.

Reply:
For now we build it for WPE-Android because of other dependencies from gstreamer

Reply:
But patch is now update according to this suggestion